src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -289,7 +289,7 @@ public static class PermutationFeatureImportanceExtensions
         /// <param name="catalog">The clustering catalog.</param>
         /// <param name="predictionTransformer">The model on which to evaluate feature importance.</param>
         /// <param name="data">The evaluation data set.</param>
-        /// <param name="labelColumnName">Label column name.</param>
+        /// <param name="labelColumnName">Label column name.  The column data must be <see cref="System.Single"/>.</param>
```

Comment:
> The column data must be <see cref="System.Single"/>. [](start = 61, length = 53)

I checked and both lightgbm ranking and fast tree should accept both single and KeyDataViewType label columns, maybe could you add that?